### PR TITLE
Migrate to Angular 6

### PR DESCRIPTION
**Pull request by luisvt**
Migrate to Angular 6

### CODE REVIEW

**Comment by Nightapes** [comment]
@luisvt can you please rebase

**Comment by luisvt** [comment]
yes, I'm on that

File: examples/src/app/forms/universal-validator/universal-validator.component.html [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-        <md-card-content>
-            <md-input-container>
-                <input mdInput type="text" [(ngModel)]="model.whitespace" name="whitespace" #formControl="ngModel" noWhitespace placeholder="Whitespace">
-                <md-hint *ngIf="formControl.hasError('noWhitespaceRequired')" class="error">No whitespace allowed</md-hint>
-            </md-input-container>
+    <mat-card class="card">
+        <mat-card-header>
+            <mat-card-title>Universal</mat-card-title>
+        </mat-card-header>
+        <mat-card-content>
```
Comment (Nightapes) [inline examples/src/app/forms/universal-validator/universal-validator.component.html]: Can you make that the example under each other, like the reactive-forms

File: examples/src/app/guide/guide.component.html [hunk 2]
```diff
@@ -12,39 +12,39 @@ <h1>ngx-validators
...
+            <mat-list-item *ngFor="let items of universal">
+                <h4 mat-line>{{items.name}}</h4>
+            </mat-list-item>
+            <mat-divider></mat-divider>
+            <h3 mat-subheader>Creditcards validators</h3>
+            <mat-list-item *ngFor="let items of creditcards">
+                <h4 mat-line>{{items.name}}</h4>
+            </mat-list-item>
+            <mat-divider></mat-divider>
+            <h3 mat-subheader>Phone validators</h3>
```
Comment (Nightapes) [inline examples/src/app/guide/guide.component.html]: @luisvt  Can you please remove PhoneValidators and add a link to https://github.com/Nightapes/ngx-phone-validators

File: examples/package.json [hunk 3]
```diff
@@ -13,24 +13,27 @@
...
+    "@angular/http": "^6.1.0",
+    "@angular/material": "^6.4.2",
+    "@angular/platform-browser": "^6.1.0",
+    "@angular/platform-browser-dynamic": "^6.1.0",
+    "@angular/router": "^6.1.0",
     "core-js": "^2.4.1",
-    "google-libphonenumber": "^2.0.13",
-    "ngx-validators": "file:///C:\\Users\\sbeis\\Documents\\GitHub\\ng2-validators\\ngx-validators-3.0.0.tgz",
-    "rxjs": "^5.2.0",
+    "google-libphonenumber": "^3.1.11",
```
Comment (Nightapes) [inline examples/package.json]: You can remove google-libphonenumber

File: examples/src/app/forms/universal-validator/universal-validator.component.html (hunk 1 above)
Comment (luisvt) [reply examples/src/app/forms/universal-validator/universal-validator.component.html]: done

**Comment by Nightapes** [comment]
@luisvt  thanks for the help

**Pull request merged**